istioctl/cmd/root.go

```diff
@@ -229,6 +225,24 @@ debug and diagnose their Istio mesh.
 	postInstallCmd.AddCommand(Webhook())
 	experimentalCmd.AddCommand(postInstallCmd)
 
+	analyzeCmd := Analyze()
+	hideInheritedFlags(analyzeCmd, "istioNamespace")
+	rootCmd.AddCommand(analyzeCmd)
+
+	convertIngressCmd := convertIngress()
+	hideInheritedFlags(convertIngressCmd, "namespace", "istioNamespace")
+	rootCmd.AddCommand(convertIngressCmd)
+
+	dashboardCmd := dashboard()
+	hideInheritedFlags(dashboardCmd, "namespace", "istioNamespace")
+	rootCmd.AddCommand(dashboardCmd)
+
+	proxyStatusCmd := statusCommand()
```

Comment:
`proxy-status` uses the Istio Namespace to find control plane pods.  See _proxystatus.go_:85.